### ISSUE TITLE
[Question]Crops from same HD image as a batch to utilize multi-gpus

### ISSUE THREAD

**Issue by faniuy** [junyanz/pytorch-CycleGAN-and-pix2pix#1074]
I've tried to use multi-gpu to train CycleGan using instance norm, but it seemed that the result is significantly worse than using instance norm with batch_size=1.
However, images in my dataset are very HD images which needs to be cropped in order to fit into VRAM. So, if I fill a batch with crops all from the same image, will the instance norm in theory be the same as when batch_size=1?
Thank you.

**Comment by junyanz** [comment]
It might be different. If you crop a patch, your features will be normalized based on the mean/var statistics of each patch rather than the entire image.
